extras/flake-rust.nix

```diff
@@ -7,18 +7,36 @@ let
   };
   craneLib = crane.lib.${pkgs.system}.overrideToolchain rustWithTools;
 
+  cleanSrc = craneLib.cleanCargoSource (craneLib.path src);
+
   # Library source code with extra dependencies attached
-  fullSrc = pkgs.stdenv.mkDerivation {
-    src = craneLib.cleanCargoSource (craneLib.path src);
-    name = "lbf-rust-build-env";
-    unpackPhase = ''
-      mkdir $out
-      cp -r $src/* $out
-      cd $out
-      ${copyExtraSources}
-      ${copyData}
-    '';
-  };
+  fullSrc =
```

Comment:
why not; buildEnv? as is the name of the derivation. make sense imo

Reply:
Also please expose this as one `package` because we can literally use this to export a non-nix Rust Cargo 'bundle' that has everything needed to build.